core/client/src/light/blockchain.rs

```diff
@@ -144,9 +144,20 @@ impl<S, F, Block> BlockchainHeaderBackend<Block> for Blockchain<S, F> where Bloc
 }
 
 impl<S, F, Block> BlockchainBackend<Block> for Blockchain<S, F> where Block: BlockT, S: Storage<Block>, F: Fetcher<Block> {
-	fn body(&self, _id: BlockId<Block>) -> ClientResult<Option<Vec<Block::Extrinsic>>> {
-		// TODO: #1445 fetch from remote node
-		Ok(None)
+	fn body(&self, id: BlockId<Block>) -> ClientResult<Option<Vec<Block::Extrinsic>>> {
+		let header = match self.header(id) {
+			Ok(Some(header)) => header,
+			_ => {
+				return Ok(None)
+			}
+		};
+
+		self.fetcher().upgrade().ok_or(ClientError::NotAvailableOnLightClient)?
+			.remote_body(RemoteBodyRequest {
+				header,
+				retry_count: None,
+			})
+			.into_future().wait()
```

Comment:
We should really return a `Future` instead of blocking.

Reply:
Yeah I agree, but this is a part of the `trait Backend` impl which doesn't support futures. I guess this relies on `http timeouts` instead?! 

However, this is out-of-scope of this PR and could probably refactored to be async instead.